src/run_releasenotes_check.py

```diff
@@ -188,11 +207,19 @@ def create_urls_file_if_not_exists() -> None:
             for item in temp_content:
                 outfile.write(markdown(item))
 
+    with open(RELEASE_NOTE_MD_PATH, 'r') as f:
+        html_content = f.read()
+
+    markdown_content = markdownify.markdownify(html_content, heading_style="ATX")
```

Comment:
So we basically still write as html then restore it back to md?
If so, do we think there is a case where we want user to choose if they want html or md output with a param?

Thanks.

Reply:
Lets follow one standard format in the code. 
If a user wants to convert, they can do it from command line tools.

Reply:
Got it.